contracts/Blacklist.sol

```diff
@@ -0,0 +1,33 @@
+pragma solidity ^0.4.17;
+
+import "./BlacklistInterface.sol";
+
+contract Blacklist is BlacklistInterface {
+
+    address public owner;
+
+    mapping (bytes32 => bool) public blacklisted;
```

Comment:
Either this should be private, or `isBlacklisted` should be omitted, since they both serve the same purpose.

Reply:
@Arachnid removed the public part